tools/Performance/DynamoPerformanceTests/ResultsComparer.cs

```diff
@@ -290,7 +290,11 @@ private void WriteResultsToCSV(IEnumerable<BenchmarkComparison> comparisons, str
 
             // Check that columns we care about exist
             var columnNames = new string[] { "Method", "Graph", "Mean", "Error", "StdDev" };
-            CheckCSVHeader(header, columnNames, csvPath);
+            var missingColumns = columnNames.Where(c => !header.Contains(c));
+            if (missingColumns.Count() > 0)
```

Comment:
Use `Any` as it is generally quicker as opposed to `Count` which has to enumerate the collection: `if(missingColumns.Any()) {...}`